fuseutil/fuseutil.go

```diff
@@ -1,7 +1,7 @@
-package fuseutil // import "bazil.org/fuse/fuseutil"
+package fuseutil // import "github.com/suffuse/fuse/fuseutil"
```

Comment:
It seems that each file has a reminder on how to import that specific package. If you're forking, I recommend you remove it. If someone does do not know how to import things, they will probably not be looking at the source.